yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -1447,6 +1500,7 @@ def pixelize_sph_kernel_arbitrary_grid(np.float64_t[:, :, :] buff,
     kernel_func = get_kernel_func(kernel_name)
 
     with nogil:
+        # TODO make this parallel without using too much memory
```

Comment:
Just to be clear, is this a leftover that you forgot to address or are you planning to address it in the future ?

Reply:
I plan to address it in the future, provided that I'll figure out how to do that...

Reply:
Alright, just checking !